README.md

```diff
@@ -26,9 +26,7 @@ You should install it somewhere in your `$PATH`. For Linux and macOS, a good pla
 For those with Homebrew, you'll need to `brew tap mike-engel/jwt-cli` repo to install it.
 
 ```sh
-# Tap and install jwt-cli
-brew tap mike-engel/jwt-cli
-brew install jwt-cli
+brew install mike-engel/homebrew-jwt-cli/jwt-cli
```

Comment:
I think the `homebrew-` part can be removed from here. Recently i've used `brew install mike-engel/jwt-cli/jwt-cli` and it's worked just fine!

Reply:
I suppose you can  squash commits on merge. If not, I can squash them.